Pipfile

```diff
@@ -18,6 +18,7 @@ flake8 = "==3.7.9"
 flake8-bugbear = "==20.1.4"
 flake8-docstrings = "==1.5.0"
 flake8-import-order = "==0.18.1"
+flex = "==6.14.1"
```

Comment:
@marcofavorito not a fan of having to introduce this, but we need it for the swagger/openapi spec verification

Reply:
What about these projects?
-  https://github.com/p1c2u/openapi-core
- https://github.com/p1c2u/openapi-spec-validator

Also, we could check how `connexion` validates specs files and requests/responses.
- https://github.com/zalando/connexion/blob/master/setup.py#L23-L47

Reply:
Used that now, thanks for the suggestion.